samples/XCT.Sample/ViewModels/Views/GravatarImageViewModel.cs

```diff
@@ -9,17 +11,7 @@ public class GravatarImageViewModel : BaseViewModel
 		int size = 50;
 		DefaultGravatar defaultGravatar = DefaultGravatar.MysteryPerson;
 
-		public DefaultGravatar[] Defaults { get; } = new[]
-		{
-			DefaultGravatar.Blank,
-			DefaultGravatar.FileNotFound,
-			DefaultGravatar.Identicon,
-			DefaultGravatar.MonsterId,
-			DefaultGravatar.MysteryPerson,
-			DefaultGravatar.Retro,
-			DefaultGravatar.Robohash,
-			DefaultGravatar.Wavatar
-		};
+		public DefaultGravatar[] Defaults { get; } = Enum.GetValues(typeof(DefaultGravatar)).Cast<DefaultGravatar>().ToArray();
```

Comment:
Can you revert this, please? I think that idea to use all Enums explicitly is to make it easier for devs to find all possible values.